packages-web/calendar/src/components/Calendar.ts

```diff
@@ -136,8 +150,9 @@ class Calendar extends Component<CalendarProps> {
             components: {
                 toolbar: wrapToolbar({ customViews: this.getToolbarProps(), onClickToolbarButton: this.onRangeChange })
             },
+            onNavigate: this.onNavigateHandler,
```

Comment:
Will this callback pass the correct date when you have a week view and click on more button which isn't on the first day of the week?

Reply:
This is only being triggered when a day is selected if I am not mistaken.